lib/runtime/wasmer/exports_test.go

```diff
@@ -1103,10 +1104,10 @@ func TestInstance_GrandpaGenerateKeyOwnershipProof(t *testing.T) {
 	identityPubKey, _ := ed25519.NewPublicKey(identity)
 	authorityID := identityPubKey.AsBytes()
 
-	encodedOpaqueKeyOwnershipProof, err := instance.GrandpaGenerateKeyOwnershipProof(uint64(0), authorityID)
-	require.NoError(t, err)
+	res, err := instance.GrandpaGenerateKeyOwnershipProof(uint64(0), authorityID)
 	// Since the input is not valid with respect to the instance, an empty proof is returned
-	require.Empty(t, encodedOpaqueKeyOwnershipProof)
+	require.NoError(t, err)
+	require.Nil(t, res)
```

Comment:
is this renaming and moving around necessary?

Reply:
In this case I actually did think it was helpful, since to me it is misleading to call the variable encoded since it is unmarshalled. Since there is already a diff from adding the nil require check, I thought it was okay to rename. 

However res is not that descriptive so I will change to opaqueKeyOwnershipProof